source/MaterialXRender/Handlers/HwLightHandler.h

```diff
@@ -0,0 +1,111 @@
+//
```

Comment:
Remove old file.

Reply:
I renamed the file but seems it didn't do the correct thing. I'll fix.